recipe/build.sh

```diff
@@ -31,6 +31,7 @@ cmake \
     -DCMAKE_AR=${AR} \
     -DCMAKE_RANLIB=${RANLIB} \
     -DPYTHON_EXECUTABLE="${PYTHON}" \
+    -DBoost_NO_BOOST_CMAKE=ON \
```

Comment:
@pitrou @wesm @kszucs FYI, you will run into this in the next days probably. Boost now comes with its own CMakeConfig which neatly breaks everything. This is the magic variable to turn it off again.